Python/PyPlasma.cpp

```diff
@@ -44,6 +44,25 @@ ST::string PyAnyString_AsSTString(PyObject* str)
     }
 }
 
+int PyAnyString_PathDecoder(PyObject* obj, ST::string* str)
```

Comment:
I think the second argument will need to be `void*` to comply with Python's API...

Reply:
It compiles and works on my testing in Windows. That's probably because Python's doing crazy va arg magic where type safety becomes a figment of the imagination. I can change it if you prefer, though.

Reply:
I went ahead and changed the second argument to `void*`.

Reply:
I think newer versions of GCC (GCC 8?) will complain about it (our CI is only building with GCC 5, so it doesn't trigger).  So thanks for updating it.